Fix: peer review landing page

lwasser opened the pull request:
This is an attempt the make the peer review landing page on our website a bit more user focused with a narrative. 
it breaks the page into sections.

[Current status: ](https://github.com/pyOpenSci/pyopensci.github.io/pull/287#issuecomment-1839713471)

* This PR is ready to merge text wise and structurally. the redesign i think looks nice. All it needs is graphics. it can remain open until those are done which will likely be before xmas. Or we can opt to add filler graphics and merge it as is for now. which could also be ok. i can just make some basic graphics. 
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206057663441255
  - https://app.asana.com/0/0/1205973778589091
  - https://app.asana.com/0/0/1206075195584338

willingc commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
   overlay_filter: 0.6
 intro:
-  - excerpt: "Our open peer review process supports scientists getting credit for the work invested in open Python tools. It also supports standardization of packaging and improved package visibility."
+  - excerpt: "Our open peer review process supports scientists getting credit and support for the work invested in developing scientific Python tools. It also supports scientists finding vetted and maintainer software to drive their open science workflows."
```
```suggestion
  - excerpt: "Our open peer review process supports scientists getting credit and recognition for the work invested in developing scientific Python tools. It also supports scientists finding vetted and maintained software to drive their open science workflows."
```

willingc reviewed (commented):
Editing for content

willingc commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+benefits_scientists:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Scientists need trusted and vetted tools that they can easily use to support their open science workflows. Over time and through our [partnerships with domain specific communities](partners.html) our catalog of tools that scientists can trust will grow. "
```
```suggestion
    excerpt: "Scientists need trusted and vetted tools to support their open science workflows. Through our [partnerships with domain specific communities](partners.html) our catalog of trusted tools for scientists will grow. "
```

willingc replied:
@lwasser this is the link that is causing CI to error.

willingc commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+benefits_maintainers:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Peer review provides a form of academic credit that is nor currently available to scientists developing software. Further, navigating the Python packaging ecosystem can be challenging, even for seasoned Pythonistas. Our diverse community supports scientific package maintainers in their efforts to develop and build robust software."
```
```suggestion
    excerpt: "Peer review provides a form of academic credit currently unavailable to scientists developing software. Further, peer review multiplies shared knowledge, such as navigating the Python packaging ecosystem, which is challenging, even for seasoned Pythonistas. Our diverse community supports scientific package maintainers in their efforts to develop and build robust software."
```

willingc commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+peer_review:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Software peer review, similar to review of scientific papers is a process where scientists vet software code, documentation and infrastructure. pyOpenSci leads an [open peer review process](https://www.pyopensci.org/software-peer-review/our-process/how-review-works.html) run by a community of dedicated volunteers. Reviews are supportive and fully transparent with the shared goal of  ...
```
```suggestion
    excerpt: "Software peer review, similar to review of scientific papers, is a process where scientists vet software code, documentation and infrastructure. pyOpenSci leads an [open peer review process](https://www.pyopensci.org/software-peer-review/our-process/how-review-works.html) run by a community of dedicated volunteers. Reviews are supportive and fully transparent with the shared goal of improving the quality, usability and maintainability of the software that is driving open science.
```

willingc commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+
+    * Our [partnership with the Journal of Open Source software](https://www.pyopensci.org/software-peer-review/partners/joss.html) further allows scientists to get academic credit for their work through a cross-ref enabled DOI.
+
+    * Reviews are lead by diverse teams of people from different backgrounds to enhance the their quality."
```
```suggestion
    * Reviews are led by diverse teams of people from different backgrounds to enhance the feedback quality."
```

willingc commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "You don't have to chose between pyOpenSci and JOSS. We have a collaboration with
+JOSS where you can submit your package to us for review. If your package is accepted
+and if it is in scope for JOSS, it will then be fast tracked through their
```
```suggestion
and if it is in scope for JOSS, it will then be fast tracked through JOSS'
```

willingc commented on _pages/about-peer-review.md:
```diff
@@ -80,36 +155,49 @@ select them here.
 
-### Who can submit a package?
+
+<!-- ### Who can submit a package?
```
Unless you are going to use this in the next week or so, I would remove all comments as they clutter the content and can always be resurrected from the repo history.

lwasser replied:
great point - i'll do that now!!

willingc commented on _sass/minimal-mistakes/_archive.scss:
```diff
@@ -240,8 +240,8 @@
-  margin-bottom: 2em;
-  border-bottom: 1px solid $border-color;
+  margin-bottom: .02em; //used to be 2em
+  // border-bottom: 1px solid $border-color;
```
remove

willingc commented on _sass/minimal-mistakes/_pyos-grid.scss:
```diff
@@ -3,7 +3,8 @@
 .grid {
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    // min max can be used to modify width
```
```suggestion
    // minmax can be used to modify width
```

willingc commented on _sass/minimal-mistakes/_pyos-grid.scss:
```diff
@@ -85,8 +86,8 @@
     border-radius: 4px;
-    padding: 32px 24px;
-    margin: 12px;
+    // padding: 32px 24px;
```
```suggestion

```

willingc commented on _sass/minimal-mistakes/_pyos-grid.scss:
```diff
@@ -85,8 +86,8 @@
-    padding: 32px 24px;
-    margin: 12px;
+    // padding: 32px 24px;
+    // margin: 12px;
```
```suggestion
```

willingc commented on _sass/minimal-mistakes/_pyos-grid.scss:
```diff
@@ -135,20 +136,21 @@
-        transition: transform 0.15s ease-out;
-    }
+
+    // &:before {
```
Remove the block 😄

lwasser replied:
@willingc 😆 you have caught me in my usual pattern of being nervous to sometimes delete things so i comment it out until i'm sure i want to really delete it! i'll go through and cleanup comments. this was me removing the circles on our people cards but i think i want to keep them for other content cards . so i'll make that fix locally!!

willingc reviewed (commented):
Looking good Leah. Most are small content tweaks.

lwasser commented:
to view the rendered page [you can go here](https://output.circle-artifacts.com/output/job/adc7187f-0bab-41ad-8ef6-b3f6075ef9f7/artifacts/0/_site/about-peer-review/index.html) 

in the artifacts section of our circleci build!! i will swap out images with more fitting graphics once we have stable content . right now i just used the same image over and over as a placeholder.

willingc commented:
It looks great @lwasser. :sunny:

willingc reviewed (commented):
Looking good.

lwasser commented:
thank you so much @willingc will be making and integrating all of these changes in the next day.

lwasser commented:
[the related fix to the author page in our peer review guide here](https://github.com/pyOpenSci/software-peer-review/pull/263/files) 

the idea being that an author hits the peer review page and then can be directed to the author guide in the "Ready to submit" section. That guide page then has relevant sections of the guide for them to review creating an author content pathway. 

<img width="853" alt="Screenshot 2023-11-29 at 3 51 07 PM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/ed509a4e-660a-4819-aa07-f4329c2914a0">

willingc reviewed (approved):
CI is passing. I'm going to approve. @lwasser 🚢 it.

kierisi commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+benefits_maintainers:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Peer review provides a form of academic credit currently unavailable to scientists developing software. Further, peer review multiplies shared knowledge, such as navigating the Python packaging ecosystem, which is challenging, even for seasoned Pythonistas. Our diverse community supports scientific package maintainers in their efforts to develop and build robust software."
```
```suggestion
    excerpt: The pyOpenSci peer review process multiplies shared knowledge, making it easier for Pythonistas of all levels to accomplish challenging tasks, such as navigating the Python packaging ecosystem, with relative ease. And our diverse community supports scientific package maintainers in their efforts to develop and build robust software."
```

kierisi commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Peer review provides a form of academic credit currently unavailable to scientists developing software. Further, peer review multiplies shared knowledge, such as navigating the Python packaging ecosystem, which is challenging, even for seasoned Pythonistas. Our diverse community supports scientific package maintainers in their efforts to develop and build robust software."
+    title: "Benefits for open source maintainers"
```
```suggestion
    title: "pyOS software peer review benefits open source maintainers"
```

kierisi commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+peer_review:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Software peer review, similar to review of scientific papers, is a process where scientists vet software code, documentation and infrastructure. pyOpenSci leads an [open peer review process](https://www.pyopensci.org/software-peer-review/our-process/how-review-works.html) run by a community of dedicated volunteers. Reviews are supportive and fully transparent with the shared goal of ...
```
```suggestion
    excerpt: "Software peer review, similar to the review of scientific papers, is a process where scientists vet software code, documentation and infrastructure. pyOpenSci leads an [open peer review process](https://www.pyopensci.org/software-peer-review/our-process/how-review-works.html) run by a community of dedicated volunteers. Reviews are supportive and fully transparent with the shared goal of improving the quality, usability and maintainability of the software that is driving open science."
```

kierisi commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Software peer review, similar to review of scientific papers, is a process where scientists vet software code, documentation and infrastructure. pyOpenSci leads an [open peer review process](https://www.pyopensci.org/software-peer-review/our-process/how-review-works.html) run by a community of dedicated volunteers. Reviews are supportive and fully transparent with the shared goal of ...
+
+    * Our [partnership with the Journal of Open Source Software](https://www.pyopensci.org/software-peer-review/partners/joss.html) further allows scientists to record academic credit for their work through a cross-ref enabled DOI.
```
```suggestion
    * Our [partnership with the Journal of Open Source Software](https://www.pyopensci.org/software-peer-review/partners/joss.html) allows scientists to record academic credit for their work through a cross-ref enabled DOI.
    * ```

kierisi commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+
+    * Our [partnership with the Journal of Open Source Software](https://www.pyopensci.org/software-peer-review/partners/joss.html) further allows scientists to record academic credit for their work through a cross-ref enabled DOI.
+
+    * Reviews are led by diverse teams of people from different backgrounds to enhance the feedback quality."
```
```suggestion
    * Diverse teams lead each review, enhancing the overall feedback quality."
```

kierisi commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,98 @@
+JOSS where you can submit your package to us for review. If your package is accepted
+and if it is in scope for JOSS, it will then be fast tracked through JOSS'
+review process; they will only review your paper.md file. Consider this a
+win-win!
```
```suggestion
    excerpt: "Our collaboration with JOSS means that you don't have to chose between pyOpenSci and JOSS. Submit your package to pyOS for review, and if your package is accepted and in scope for JOSS, it will be fast-tracked through JOSS' review process; they will only review your `paper.md` file. It's a win-win!"
```

lwasser replied:
i want to commit this one but i suspect this edit needs to be applied to lines 38-42 and here it's replacing just line 42 with that entire blog of text. i'll come back to this edit !! the text is excellent!

kierisi reviewed (commented):
added some copy suggestions!

willingc commented:
Awesome @kierisi. Love the alt text additions. @lwasser I will let you rereview content and it looks good to me.

lwasser commented on _pages/about-peer-review.md:
```diff
@@ -1,46 +1,99 @@
+joss:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "You don't have to chose between pyOpenSci and JOSS. We have a collaboration with
+JOSS where you can submit your package to us for review. If your package is accepted
+and if it is in scope for JOSS, it will then be fast tracked through JOSS'
+review process; they will only review your paper.md file. Consider this a
+win-win!
```
```suggestion
    excerpt: "You don't have to chose between pyOpenSci and JOSS. We have a collaboration with
JOSS where you can submit your package to us for review. If your package is accepted
and if it is in scope for JOSS, it will then be fast tracked through JOSS'
review process; they will only review your paper.md file. Consider this a
win-win!](excerpt: "Our collaboration with JOSS means that you don't have to chose between pyOpenSci and JOSS. Submit your package to pyOS for review, and if your package is accepted and in scope for JOSS, it will be fast-tracked through JOSS' review process; they will only review your `paper.md` file. It's a win-win!"
```

lwasser replied:
```suggestion
    excerpt: "Our collaboration with JOSS means that you don't have to chose between pyOpenSci and JOSS. Submit your package to pyOS for review, and if your package is accepted and in scope for JOSS, it will be fast-tracked through JOSS' review process; they will only review your `paper.md` file. It's a win-win!"
```

lwasser commented:
ok updates!
✅ i've merged all of your input here @kierisi thank you so much for all of the content edits. I welcome a second round if you wish to look at things again
✅ i've cleaned up the styles for the page - specifically the text and block at the bottom was too wide to be readable. 

what's left

- [ ] new images for the page that align with our colors and "playful" style
- [ ] alt tags for those images

[The current rendered page can be seen here](https://output.circle-artifacts.com/output/job/cdf0c348-5015-446b-9137-dd04ff28be6f/artifacts/0/_site/about-peer-review/index.html)

@kierisi please in working on this page i realized the JOSS section had been removed so i added it back. 
I also think we could have images that are a bit less square to balance the page out some. 

i'm going to open a separate PR to update our partners page. @kierisi i suspect you'll have really good feedback for that page as well. it's a similar style but also needs some cleanup of messaging and brand. i might be able to create better diagrams now that i'm using canva more but it still has some AI generated images of people (we do want representations of people all over!!)

willingc commented:
@lwasser @kierisi May I make a small suggestion about PRs...

When making changes that are fluid (I.e. text, images, styling), it's often easier to create multiple PRs instead of combining into one PR.

I typically ask myself:
- Is this one change better than what is already there?
- Can this change stand alone without other changes?
- Is there any significant downside to one PR landing a day or two before the next PR? (I use the Issues to keep track of to do items.)

For example, I would recommend landing all of the content changes in this PR, create another PR for the images, and another for the styling.

willingc commented on _pages/about-peer-review.md:
```diff
@@ -1,152 +1,133 @@
-track your package through their process given it was already reviewed by us.
-Once accepted by JOSS, you now have both a pyOpenSci acceptance and one by JOSS.
-Joss will then give you a cross-ref supported DOI for citation.
+The pyOpenSci software peer review process is lead by a volunteer team of
```
```suggestion
The pyOpenSci software peer review process is led by a volunteer team of
```

willingc commented on _pages/about-peer-review.md:
```diff
@@ -1,152 +1,133 @@
+editors from the scientific Python community. Editors do the following things:
 
-### Why Two Review Processes JOSS and pyOpenSci?
+* They find reviewers from diverse backgrounds for each package that is submitted that have a mixture of scientific domain and Python experience.
```
```suggestion
* They find reviewers from diverse backgrounds who have a mixture of scientific domain and Python experience.
```

willingc reviewed (approved):
Small typo and clarifying text. 🚢

lwasser commented:
> @lwasser @kierisi May I make a small suggestion about PRs...
> 
> When making changes that are fluid (I.e. text, images, styling), it's often easier to create multiple PRs instead of combining into one PR.
> 
> I typically ask myself:
> 
> * Is this one change better than what is already there?
> * Can this change stand alone without other changes?
> * Is there any significant downside to one PR landing a day or two before the next PR? (I use the Issues to keep track of to do items.)
> 
> For example, I would recommend landing all of the content changes in this PR, create another PR for the images, and another for the styling.

Thank you @willingc this is helpful. i will definitely think about this more carefully for future pr's. In this case when i went to edit the page there was a lot of inline HTML with styles applied there that made the content more difficult to review. 

so i pulled that html out and tried to move all of the text to the front matter for simplified editing and then templated content via includes and styles that could be resused. it was easier for me to do this all at once but i can see how as a reviewer it would make for a more complex review.  i'll think about how to parse those elements out separately in the future. it's just a bit tricky when some of these pages were created a bit haphazardly (by me!) 

we have asana tasks locally to track updates to images. But we can't edit alt tags on the page until images are fixed. i'm open to suggestions that you have about how to chunk the work buckets out a bit more cleanly as well.

willingc reviewed (dismissed):
Let's land this PR :smile:

willingc commented:
@lwasser Looks like there is a merge conflict here.
